developers/guides/access-control-webhooks.mdx

```diff
@@ -231,6 +231,32 @@ to the m3u8 URL as a query parameter, similar to:
 https://playback.livepeer.studio/asset/hls/abcd1234/index.m3u8?accessKey=your-access-key
 ```
 
+If you prefer not to use query parameters to pass the Access Key, 
+it can alternatively be transmitted as a header. To accomplish this, 
+you must set a header named `Livepeer-Access-Key` with your access key as its value.
+
+Below is an example of how to use headers when using 
+the Livepeer Player:
+
+```tsx
+import { Player } from "@livepeer/react";
+
+const hlsConfig = {
+  xhrSetup: function(xhr, url) {
+    xhr.setRequestHeader('X-Livepeer-Access-Key', 'YOUR_ACCESS_KEY');
```

Comment:
Any reason to have the `X-` prefix here but not in the other one? Maybe this one is a typo 👀

Reply:
Yep just a typo :D